numpy/random/src/distributions/random_hypergeometric.c

```diff
@@ -1,6 +1,7 @@
+#include "Python.h"
```

Comment:
Nothing in this file should require `Python.h`

Reply:
```suggestion
```
NumPy compiles without this change on latest Cygwin.